dev/request-logging-dev.xml

```diff
@@ -0,0 +1,8 @@
+<configuration scan="true" debug="false">
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+        <pattern>%h %l %u %user %date "%r" %s %b %h %a %localPort %D</pattern>
+    </encoder>
+  </appender>
+  <appender-ref ref="STDOUT" />
+</configuration>
```

Comment:
Yeah, having the output mixed in with the other STDOUT looked pretty bad.  Will move it to ./target